### ISSUE TITLE
Issue with Custom Component with props

### ISSUE THREAD

**Issue by yaylinda** [Maronato/vue-toastification#20]
First of all, love this plugin. It's beautiful and easy to use, with great documentation! The demo page helps a lot too!

I encountered an issue where I'm trying to pass a custom component as the `ToastContent`, while passing props and listeners along with my component. 

So this works fine: `this.$toast.info(ToastActionComponent);`

However, this does not work: 
```
const toastContent = {
    component: ToastActionComponent,
    props: {
        toastType: ToastType.END_TURN,
        msgObj: msgObj // defined a few lines earlier
    }
}

this.$toast.info(toastContent);
```

I get this warning in the console:
```
vue.runtime.esm.js?2b0e:619 [Vue warn]: Failed to mount component: template or render function not defined.

found in

---> <Anonymous>
       <Toast>
         <TransitionGroup>
           <Transition>
             <Root>
```

Just to clarify, by "does not work" I mean, the Toast notification shows up as a blank Toast. The content of my `ToastActionComponent` does not get rendered.

And this is my (simplified) custom component: `ToastActionComponent`
```
// ToastActionComponent.vue

<template>
    <div>
        HI!
    </div>
</template>

<script lang="ts">
import { Component, Vue, Prop, Watch } from "vue-property-decorator";
import { ToastType } from '../utils/constants';

@Component({})
export default class ToastActionComponent extends Vue {

    @Prop() toastType!: ToastType;
    @Prop() msgObj!: any;

}

</script>

<style lang="scss" scoped>

</style>
```

**Comment by Maronato** [comment]
Hey @yaylinda! I'm glad to hear you like it! Thanks for reporting this :)
I think I know where the issue is. It is related to class-based components and since I don't usually use them it would've taken some time for me to notice.

I'll push a fix in a couple of minutes.

**Comment by Maronato** [comment]
A fix was released under [v0.5.2](https://github.com/Maronato/vue-toastification/releases/tag/v0.5.2). Please give it a go and reopen the issue if the problem remains

**Comment by yaylinda** [comment]
Thanks @Maronato ! I Will try it out :)
